dev/integration_tests/ios_host_app/flutterapp/lib/main

```diff
@@ -7,6 +7,7 @@ import 'dart:ui' as ui;
 
 import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
+// UNCOMMENT: import 'package:ffi_package/ffi_package.dart';
```

Comment:
> Because the test harness checks that add-to-app works by clicking buttons, so all that logic in the main file needs to be there.

We wouldn't need to re-run those same tests a second time though. I'm saying we can build a project, run the existing tests, replace main, and then run different, FFI-specific tests. Am I missing something about the flow that makes that infeasible?

Reply:
There are two sections before all the plugins are started:

```dart
section('Build ephemeral host app in release mode without CocoaPods');

// ... run a `flutter build`

// Check the tool is no longer copying to the legacy xcframework location.
checkDirectoryNotExists(path.join(projectDir.path, '.ios', 'Flutter', 'engine', 'Flutter.xcframework'));
```

And some extra checks that the build succeeded.

Maybe we can just move those checks until after the extra plugins/packages are added. I'll take a look.

Reply:
I have moved the checks until after the native assets package and other plugins are added. The `// UNCOMMENT: ` logic has been removed.